controller/ControllerRTL.py

```diff
@@ -102,6 +102,7 @@ def construct(s, ControllerIdType, CmdType, CtrlPktType, NocPktType,
       assert addr2controller_vector[addr_base] == -1, f"address range [{begin_addr}, {end_addr}] overlaps with others."
       addr2controller_vector[addr_base] = ControllerIdType(src_controller_id)
 
+      # What does this do? Connect itself?

```

Comment:
This is to model/construct a look-up table (i.e., a map, a bunch of wires in HW) at design time to memorize the relationship between `addr_base` and `controller_id`. 

Theoretically, this `addr2controller_vector` should also be delivered via `cpu_pkt`, and the look-up table should be updated accordingly. Can you please help create/file an issue for this?